09-PowershellModule/Readme.md

```diff
@@ -0,0 +1,128 @@
+
+## Installation Guide
+
+>**Pre-requisites**:
+> - PowerShell 7.0 or higher. 
+> - Windows OS
+	
+1. First verify that prerequisites are already installed:  
+    Ensure that you are using Windows OS and have PowerShell version 7.0 or higher by typing **$PSVersionTable** in the PS/ISE console window and looking at the PSVersion in the output as shown below.) 
+    <br><br>
+ ![PowerShell Version](../Images/09_PS_VERSION.png) 
+ <br><br>
+ If the PSVersion is older than 7.0, update PowerShell using command     
+```Powershell
+    iex "& { $(irm aka.ms/install-powershell.ps1) } -UseMSI"
+```
+
+2. Install the Tenant Security Solution for Azure (AzTS) PS module:  
+	  
+```PowerShell
+  Install-Module AzTS -Scope CurrentUser
+```
+
+
+## AzTS command
+
+> **Note**: AzTS cmdlet support 3 letter acronyms (e.g.: IAS). You can invoke cmdlet using these *after* AzTS has been imported in the session. So, to use this alias, make sure you run 'ipmo AzTS' as the first thing in a new PS/ISE console window ('ipmo' itself is an alias for 'import-module'). Apart from cmdlet, parameters also have associated aliases.
+
+| Command (alias) | What it does |	Required Permission |
+|----|----|-----|
+|Invoke-AzureScan (IAS)|Scans an Azure subscription for security best practices and configuration baselines for things such as alerts, ARM policy, RBAC, MDC, etc.|Reader on subscription|
+
+
+
+## List of commonly used parameters
+
+| Parameter (alias) |
+|-------------------|
+|SubscriptionId (sid)|
+|TenantId (tid)|
+|FilterTags (ftg)|
+|ResourceTypeNames (rtn)|
+|ExcludeResourceTypeNames (ertn)|
+|ControlIds (cids)|
+|ExcludeControlIds (ecids)|
+
+----------------------------------------------------------
+## AzTS: Security scanner for Azure
+
+>  **Prerequisites:**
+> For all commands in this feature it is assumed that you have:
+> 1. Logged in to your Azure account using Connect-AzAccount from a PowerShell ISE.
+> 2. Selected a subscription using Set-AzContext.
+
+### Overview
+ 
+Security Scanner for Azure scans all the prominent features in Azure (e.g., Web Apps, Storage, SQL DB, Key Vault, etc.) by
+executing the security controls and will validate their status as 'Success' or 'Failure' based on the security guidance.
+
+The following aspects of security are also checked:
+1. Access control configuration - identity and access management related issues in the subscription.
+2. Alert configuration - configuration of activity alerts for sensitive actions for the subscription and various cloud resources.
```

Comment:
This activity alert is not there in AzTS yet.

Reply:
Done